experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { EventEmitter } from "events";
+import { IAudience } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import {
+    LoadableObjectClass,
+} from "./types";
+import { RootDataObject } from "./rootDataObject";
+
+export class FluidContainer
+    extends EventEmitter
+    implements
+        Pick<Container, "audience" | "clientId" | "close" | "closed" | "connected" >,
+        Pick<RootDataObject, "initialObjects"> {
+    private readonly _container: Container | undefined = undefined;
+    private readonly _rootDataObject: RootDataObject | undefined = undefined;
+
+    public constructor(container: Container, dataObject: RootDataObject) {
+            super();
+            this._container = container;
+            this._rootDataObject = dataObject;
+
+            container.on("connected", (id: string) =>  this.emit("connected", id));
+    }
+
+    static async create(container: Container) {
+        const rootDataObject = (await container.request({ url: "/" })).value;
+        return new FluidContainer(container, rootDataObject);
+    }
+
+    private get container(): Container {
+        assert(this._container !== undefined, "container undefined");
+        return this._container;
+    }
+
+    private get rootDataObject(): RootDataObject {
+        assert(this._rootDataObject !== undefined, "defaultObject undefined");
+        return this._rootDataObject;
+    }
+
+    public async create<T extends IFluidLoadable>(objectClass: LoadableObjectClass<T>): Promise<T> {
+        return this.rootDataObject.create(objectClass);
+    }
+
+    public close() {
+        this.container.close();
+    }
+
+    public get initialObjects() {
+        return this.rootDataObject.initialObjects;
+    }
+
+    public get closed() {
+        return this.container.closed;
+    }
+
+    public get connected() {
+        return this.container.connected;
+    }
+
+    public get audience(): IAudience {
+        return this.container.audience;
+    }
+
+    public get clientId() {
+        return this.container.clientId;
+    }
```

Comment:
This comment is out of scope for this PR but still worth asking.

@sumedhb1995 / @micahgodbolt do you see clientId and audience living on FluidContainer? It seems like we are going in a service specific direction which would place these properties outside of the FluidContainer

Reply:
Yes, I was thinking of these values as something that would live on the additional container services object that would be returned alongside the FluidContainer itself.

Reply:
Something along these lines: https://github.com/microsoft/FluidFramework/pull/6074/files#diff-7ba4ac43f91ab53a0c43be6df9f145fdfa4e5a9b109254b7da57304379dcf71fR49

Audience would be TinyliciousClient's first container service. OdspClient already has a container services object that currently just holds the generateLink callback. Audience would be another value on to that services object
https://github.com/microsoft/FluidM365Integration/pull/17/files#diff-f42163c87eeb19e539e51ec0eeff0b54e87145ea26b651bdc7592f2e1ffddbd1R100

Reply:
ok, happy to remove that